catalog/urls.py

```diff
@@ -1,12 +1,15 @@
 from django.urls import path
 
-from .views import HomeListView, ContactFormView, ProductDetailView, ProductCreateView
+from .views import ProductListView, ContactFormView, ProductDetailView, ProductCreateView, ProductUpdateView, \
+    ProductDeleteView
 
 app_name = 'catalog'
 
 urlpatterns = [
-    path("home/", HomeListView.as_view(), name="home"),
     path("contacts/", ContactFormView.as_view(), name="contacts"),
-    path("product_info/<int:pk>/", ProductDetailView.as_view(), name='product_detail'),
-    path("add_product/", ProductCreateView.as_view(), name="add_product"),
+    path("products/", ProductListView.as_view(), name="home"),
+    path("products/<int:pk>/product_info/", ProductDetailView.as_view(), name='product_detail'),
+    path("products/add_product/", ProductCreateView.as_view(), name="add_product"),
+    path("products/<int:pk>/product_info/edit/", ProductUpdateView.as_view(), name='product_edit'),
```

Comment:
Можно не указывать `product_info` - это лишь удлиняет строку, без необходимости. Контекст понятен и так, ведь ты передаешь id сущности
```suggestion
    path("products/<int:pk>/edit/", ProductUpdateView.as_view(), name='product_edit'),
```
В остальных url тоже